src/Symfony/Component/DependencyInjection/Tests/Compiler/AutowirePassTest.php

```diff
@@ -900,67 +885,101 @@ public function __construct($foo, Bar $bar, $baz)
     }
 }
 
-class SetterInjection
+class SetterInjection extends SetterInjectionParent
 {
+    /**
+     * @required
+     */
     public function setFoo(Foo $foo)
     {
         // should be called
     }
 
+    /** @inheritdoc*/
```

Comment:
It lacks a test for a child method with no dock block and a parent method marked with `@required`.